clang/docs/analyzer/checkers.rst

```diff
@@ -938,6 +938,53 @@ optin.portability.UnixAPI
 """""""""""""""""""""""""
 Finds implementation-defined behavior in UNIX/Posix functions.
 
+.. _optin-taint-TaintAlloc:
+
+optin.taint.TaintAlloc (C, C++)
```

Comment:
I was thinking of suggesting `TaintedAlloc` (same in past tense), as it sounds better to me. I have no strong opinion on this though.